orte/mca/grpcomm/grpcomm.h

```diff
@@ -123,6 +125,15 @@ typedef int (*orte_grpcomm_base_module_xcast_fn_t)(orte_vpid_t *vpids,
 typedef int (*orte_grpcomm_base_module_allgather_fn_t)(orte_grpcomm_coll_t *coll,
                                                        opal_buffer_t *buf);
 
+/* Reliable broadcast a message thru BMG.
+ * only need to provide a message buffer, dont need create dmns
+ */
+typedef int (*orte_grpcomm_base_module_rbcast_fn_t)(opal_buffer_t *msg);
```

Comment:
What is the difference with the callback you defined few lines above ?

Reply:
This is reliable broadcast send, before is a callback function for receiving, other module(propagator) can register this callback function.